lib/docker/image.go

```diff
@@ -4,11 +4,32 @@ import (
 	"io"
 	"os"
 	"os/exec"
+	"strings"
 
 	"github.com/docker/docker/api/types"
+	"github.com/sdslabs/gasper/lib/utils"
 	"golang.org/x/net/context"
 )
 
+// Check for available images and pull if not present
+func CheckAndPullImages(imageList ...string) {
```

Comment:
any reason why to migrate this here from helper?

Reply:
Yes,
1. This function is specific to docker so it should be in docker package only, I think.
2. I had to reuse the above function and can't import functions from global files but packages. So, I thought better to migrate.